src/main/java/com/Novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,56 @@
+package com.Novi.TechItEasy.controllers;
+
+import com.Novi.TechItEasy.exceptions.NameTooLongException;
+import com.Novi.TechItEasy.exceptions.RecordNotFoundException;
+import jakarta.websocket.server.PathParam;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestController
+public class TelevisionController {
+
+    String[] televisionBrands = {"Sony", "Phillips", "LG"};
+    List<String> televisionDatabase = new ArrayList<>(List.of(televisionBrands));
+
+
+
+    @GetMapping(value ="/televisions")
+    public ResponseEntity<List<String>> getTelevisions() {
+        return ResponseEntity.ok(televisionDatabase);
+    }
+
+    @GetMapping(value ="/television/{id}")
+    public ResponseEntity<String> getTelevision(@PathVariable int id) {
+        if (id >= televisionDatabase.size()) {
+            throw new IndexOutOfBoundsException();
+        } else{
+            return ResponseEntity.ok(televisionDatabase.get(id));
+        }
+    }
+
+    @PostMapping(value ="/television/add")
```

Comment:
Je moet geen "add" gebruiken in je paths. Het feit dat het een POST methode is, maakt al duidelijk dat het om een add-achtige functie gaat.